bin/release.php

```diff
@@ -0,0 +1,158 @@
+<?php
+// phpcs:ignoreFile
```

Comment:
I have few questions regarding this file

1. Any particular reason of doing this in PHP? I think writing a bash script will be more appropriate.
2. Instead of using `phpcs:ignoreFile` I think we should ignore lines in which we are facing false error (if we are using bash script then this issue shouldn't occur)
3. Can we also add this file in PHPStan by updating configs from `phpstan.neon` (if we are not upgrading to bash script)

Reply:
> Any particular reason of doing this in PHP? I think writing a bash script will be more appropriate.

I have no experience with bash, and wanted to get this done quickly. So writing this in PHP makes this faster to code, maintain and amend, at least for me. Developers who join the project might not know bash, but they will surely know PHP. So PHP makes sense to me. It could definitely be converted to bash in the future if there's a good reason for that, but it's not something that I'm planning in this first iteration.

> Instead of using phpcs:ignoreFile I think we should ignore lines in which we are facing false error (if we are using bash script then this issue shouldn't occur)

There were _lots_ of lines with errors, so picked this approach. I don't think there is a major reason to ignore specific lines, since this is not really a part of the plugin code.

> Can we also add this file in PHPStan by updating configs from phpstan.neon (if we are not upgrading to bash script)

Good idea. Will do 🙂